projects/packages/publicize/src/rest-api/class-connections-post-field.php

```diff
@@ -329,7 +329,7 @@ protected function get_meta_to_update( $requested_connections, $post_id = 0 ) {
 		foreach ( $changed_connections as $id => $enabled ) {
 			$connection = $available_connections_by_connection_id[ $id ];
 
-			if ( $connection['done'] || ! $connection['toggleable'] ) {
+			if ( $connection['done'] ) {
```

Comment:
Is `$connection['done']` present? I don't know so. We don't return it from `get_filtered_connection_data`. Should we add it there?

Reply:
Yeah looks like we removed it here https://github.com/Automattic/jetpack/commit/239a66046b8f766e6d613176a78917115192aa5a#diff-540cd16120af78cc4afc265ca572d7dac4674ebd81ec12f9479bf079929dc04aL1014 so we should probably add that back in, although I think `done` is determined by the post being published. It would be strange to always skip over changes to that.

Reply:
I added it back in a922acd. I think that's used to avoid doing multiple updates

Reply:
It's that connections are always disabled on published posts. It kind of makes sense as we only attempt to share to the connections as we publish the post for the first time, but it seems a little odd.

We will probably need to look at that with the resharing, although I think what connections get shared to, are stored elsewhere for that.